traffic_portal/app/src/common/modules/locks/locks.tpl.html

```diff
@@ -0,0 +1,23 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<div id="locksContainer">
+    <div class="alert alert-dark" ng-repeat="l in locks">
+        <button type="button" class="close" title="Unlock {{l.cdn}}" aria-hidden="true" ng-click="confirmUnlock(l)"><i class="fa fa-unlock"></i></button>
+        <div ng-bind-html="l.cdn + ' is ' + ((l.soft) ? 'soft' : 'hard') + ' locked by ' + l.userName + ' (' + l.message + ')' | linky:'_blank'"></div>
```

Comment:
Why use `ng-bind-html` instead of just a regular template?

Reply:
because of the linky directive

Reply:
I guess you can't do like `{{l.message | linky:_blank}}`?

Reply:
i can try. tbh, i just copied this from notifications.tpl.html which was copied from messages.tpl.html :)

Reply:
I couldn't find any examples anywhere that didn't use `ng-bind-html`, so it probably won't work without it. But I didn't actually try it.